tests/TestCase.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Akaunting\Tests\Money;
```

Comment:
The namespace must be `Akaunting\Money\Tests`. Same issue for all test classes.

Reply:
Fixed.